pom.xml

```diff
@@ -337,12 +231,12 @@
   <properties>
     <revision>1.7.1</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.176.4</jenkins.version>
+    <jenkins.version>2.243</jenkins.version>
```

Comment:
We definitely can't take this plugin to this as the minimum version.

Reply:
That’s fair enough - I fully expect this PR to stay open until there are other reasons to go to a version after 2.243.